policy/schema.py

```diff
@@ -338,6 +345,20 @@ def on_policy_mutation(sender, **kwargs):
             policy=policy, mutation_id=kwargs['mutation_log_id'])
     return []
 
+def on_policy_renewal_mutation(sender, **kwargs):
+    uuids = kwargs['data'].get('uuids', [])
+    if not uuids:
+        uuid = kwargs['data'].get('policy_renewal_uuid', None)
+        uuids = [uuid] if uuid else []
+    if not uuids:
+        return []
+    impacted_policy_renewals = PolicyRenewal.objects.filter(uuid__in=uuids).all()
+    for policy_renewal in impacted_policy_renewals:
+        PolicyRenewalMutation.objects.create(
```

Comment:
There is an object_mutated helper for this but this works too.

Reply:
should I use it as it is at `gql_mutations.py#CreateRenewOrUpdatePolicyMutation.do_mutate#L51`?

```python
PolicyMutation.object_mutated(user, client_mutation_id=client_mutation_id, policy=policy)
```